plugins/uefi-capsule/fu-self-test.c

```diff
@@ -264,8 +277,11 @@ main (int argc, char **argv)
 	g_test_add_func ("/uefi/bgrt", fu_uefi_bgrt_func);
 	g_test_add_func ("/uefi/framebuffer", fu_uefi_framebuffer_func);
 	g_test_add_func ("/uefi/bitmap", fu_uefi_bitmap_func);
+	/* on linux ESRT data can be faked easily */
+#ifdef __linux__
```

Comment:
i think still compile the tests, but ifndef __linux__ then do a `g_test_skip` and then `return` at the start.